plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -84,6 +84,7 @@ var _ admission.MutationInterface = &PodSecurityPolicyPlugin{}
 var _ admission.ValidationInterface = &PodSecurityPolicyPlugin{}
 var _ genericadmissioninit.WantsAuthorizer = &PodSecurityPolicyPlugin{}
 var _ kubeapiserveradmission.WantsInternalKubeInformerFactory = &PodSecurityPolicyPlugin{}
+var fullPluginName = strings.ToLower(PluginName) + admission.AnnotationSuffix
```

Comment:
admissionPluginDomain. Can we standardize on some schema? `lowercase(pluginname).admission.k8s.io`?

Reply:
nvmd. It's exactly what the code is doing 👍

Reply:
just the var name can be improved.

Reply:
done